canvases/dark-packages/main.dark

```diff
@@ -254,4 +254,12 @@ let _handler _req =
     | Some Module -> "module"
     | None -> "not found"
 
-  PACKAGE.Darklang.Stdlib.Http.response (respBody |> String.toBytes) 200
\ No newline at end of file
+  PACKAGE.Darklang.Stdlib.Http.response (respBody |> String.toBytes) 200
+
+
+
+[<HttpHandler("GET", "/openai-chat/:prompt")>]
```

Comment:
It seems like this HTTP handler doesn't belong in this canvas. Maybe we should move it to a separate canvas?

Reply:
I'm not sure why we need this endpoint - why not do the `(PACKAGE.OpenAI.ChatCompletion.completion prompt) |> unwrap` in the CLI code?

Reply:
In the CLI, we use `CLI.HttpClient.request`, while `PACKAGE.OpenAI.ChatCompletion.completion` uses `HttpClient.request`. Attempting to use it directly results in a 'Function HttpClient.request is not found' error. This is why I needed to add the endpoint.

Reply:
Yes, i agree with Stachu. I guess we need to detangle HttpClient.request from LibBackend.